terra-jupyter-base/Dockerfile

```diff
@@ -54,10 +53,15 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     git \
     jq \
 
-# for jupyterlab extensions
+    # for jupyterlab extensions
     nodejs-dev \
     node-gyp \
     npm \
+
+    # java
+    default-jre \
```

Comment:
```
(base) jupyter-user@310f034381c2:~$ java -version
openjdk version "11.0.5" 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-post-Ubuntu-0ubuntu1.118.04)
OpenJDK 64-Bit Server VM (build 11.0.5+10-post-Ubuntu-0ubuntu1.118.04, mixed mode, sharing)
```

Reply:
I actually downgraded the Hail image to JDK8 because that's what was previously installed. 

@kyuksel raised the question about whether we should peg the version here. I'm not sure -- but I kind of like just installing the Ubuntu-default version of Java in the base image, and pegging the version if any particular tool (like Hail) requires a certain Java version.

Reply:
sounds good..maybe add a comment this is jdk 11?

Reply:
My concern would be unexpected breakages due to version changes but not a blocker to me in this case.